playground/frontend/lib/components/dropdown_button/dropdown_button.dart

```diff
@@ -88,8 +102,16 @@ class _AppDropdownButtonState extends State<AppDropdownButton>
             alignment: WrapAlignment.center,
             crossAxisAlignment: WrapCrossAlignment.center,
             children: [
+              widget.leading != null
+                  ? Padding(
+                      padding: const EdgeInsets.only(right: kMdSpacing),
+                      child: widget.leading,
+                    )
+                  : const SizedBox(),
```

Comment:
There's a good habit to put the shortest branch of a condition first, it's easier to read.

Reply:
Fixed.